test/get_test.js

```diff
@@ -3,7 +3,7 @@ goog.setTestOnly();
 
 const {DataLayerHelper} = goog.require('helper');
 
-describe('The helper.get function', function() {
+describe(`The 'get' function of helper`, () => {
   const h = new DataLayerHelper([{
```

Comment:
Not your change, but just noticed this. Can you add a parameter name comment?
https://google.github.io/styleguide/jsguide.html#formatting-param-name-comments

Can be in a separate PR. It would be good to audit all tests for this.

Reply:
I added the comment to this file, but didn't see anything in the others.

Reply:
There are some methods that could benefit from this. Since the same function is just used again and again tens of times, I  only put the comment in the first function of any file